src/Ledger/Utxo.lagda

```diff
@@ -49,7 +58,10 @@ utxoEntrySizeWithoutVal : MemoryEstimate
 utxoEntrySizeWithoutVal = 8
 
 utxoEntrySize : TxOut → MemoryEstimate
-utxoEntrySize utxo = utxoEntrySizeWithoutVal + size (getValue utxo)
+utxoEntrySize o = utxoEntrySizeWithoutVal + size (getValue o)
+
+refScripts : Tx → UTxO → ℙ Script
+refScripts tx utxo = ∅ -- TODO: implement when we do Babbage
```

Comment:
I think it would be good adding a documentation saying that this is only the scripts that are actually used by the transaction, i.e. native scripts and plutus scripts that have a script purpose.